### PR TITLE
EVM-C v0.2 "pylimorphic"

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: libevm/JitVM.cpp [hunk 1]
```diff
@@ -284,6 +296,7 @@ class EVM
 
 	private:
 		evm_result m_result;
+        evm_release_result_fn m_release;
```
Comment (bobsummerwill) [inline libevm/JitVM.cpp]: tab/spaces wrong?

**Comment by bobsummerwill** [comment]
Looks good to me.

@axic?

**Comment by axic** [comment]
Looks good. We need to come up with a good way to make this code reusable in cpp-ethereum. Right now I only need to change `evmjit_get_fn_table` to `hera_get_fn_table`, but perhaps in the future evmijt wants to deviate. Still, there will be a lot of shared code, such as the query/call/update bindings.

**Comment by chfast** [comment]
No problem with generalizing JitVM class -- it can take pointer `<vm-name_get_interface()` as an argument.

**Pull request merged**